starter-code/js/tenSecondsMath.js

```diff
@@ -1,6 +1,54 @@
 // Use this file to write the logic of your game, the needed attrs and functions
+function TenSecondsMathGame (){
+  this.operation='+';
+  this.number=10;
+}/* first we iniliazed and then we will modify*/
+var newGame=new TenSecondsMathGame();
 
-var TenSecondsMathGame = function(options) {
+function randomNumber(){
+  var selectedNumber=100;
+  return Math.floor(Math.random()*(selectedNumber-1)*1);
+}
+
+function variables(){
+  var number1 =randomNumber();
+  var number2= randomNumber();
+  var array=[];
+  array.push(number1,number2);
+  if (array[0]<array[1]){
+    /*it will and order to permute the values*/
+  }
+  /*var initresult=Math.abs(number1-number2);*/
+
+  if (initresult < 3 && number1%number2==0){
```

Comment:
What is 'initresult'?

Maybe, is better to check divisibility only for the division, what do you think?